### PR TITLE
Convert `metadata.json` to recipe templates

### PR DESCRIPTION
**Pull request by forana**
This PR converts the `metadata.json` files added in #304 into recipe templates - fragments of an image recipe file - with the idea that our release automation will autogenerate the rest, with the combined result made available for Saturn installations to consume.

The information lost from this conversion (OS, URI, and the github link, primarily) will all be populated by release automation.

### CODE REVIEW

File: saturn-geospatial/recipe-template.json [hunk 1]
```diff
@@ -0,0 +1,6 @@
+{
+    "name": "saturn-geospatial",
+    "description": "A Python image containing libraries specific to geospatial IO, analysis, and visualization. Generally used for CPU instances.",
+    "hardware_type": "cpu",
+    "supports": ["jupyterlab", "dask"]
```
Comment (jsignell) [inline saturn-geospatial/recipe-template.json:5]: did we decide on jupyterlab vs just jupyter?

File: saturn-pytorch/recipe-template.json [hunk 2]
```diff
@@ -0,0 +1,6 @@
+{
+    "name": "saturn-pytorch",
+    "description": "An image for deep learning with PyTorch. Includes all requirements for using PyTorch on a GPU and with Dask.",
```
Comment (jsignell) [inline saturn-pytorch/recipe-template.json:3]: This is stylistic, but do we really want to start all these with "An image for..."? This feels snappier to me:

```suggestion
    "description": "For deep learning with PyTorch. Includes all requirements for using PyTorch on a GPU and with Dask.",
```

File: saturn-rstudio/recipe-template.json [hunk 3]
```diff
@@ -0,0 +1,6 @@
+{
+    "name": "saturn-rstudio",
+    "description": "A basic image for analysis using R and RStudio.",
```
Comment (jsignell) [inline saturn-rstudio/recipe-template.json:3]:
```suggestion
    "description": "For basic analysis using R and RStudio.",
```

File: saturn/recipe-template.json [hunk 4]
```diff
@@ -0,0 +1,6 @@
+{
+    "name": "saturn",
+    "description": "A basic Python image for data analysis, machine learning, and parallel processing with Dask.",
```
Comment (jsignell) [inline saturn/recipe-template.json:3]:
```suggestion
    "description": "For basic Python data analysis, machine learning, and parallel processing with Dask.",
```

**Review by jsignell — COMMENTED** [review]
The only info we are losing is the "os" but I'm super inclined to think that is not very interesting AND it will probably be nested in the base image definition.

File: saturn-geospatial/recipe-template.json (hunk 1 above)
Comment (forana) [reply saturn-geospatial/recipe-template.json:5]: We have not - I proposed it in https://github.com/saturncloud/recipes/pull/36

**Comment by forana** [comment]
> The only info we are losing is the "os" but I'm super inclined to think that is not very interesting AND it will probably be nested in the base image definition.

It'll be available as the `base_image_uri` from each `version` on the base images.

File: saturn-geospatial/recipe-template.json (hunk 1 above)
Comment (jsignell) [reply saturn-geospatial/recipe-template.json:5]: oh I just totally missed that. Ok I think it's fine this way for now.

**Review by jsignell — COMMENTED** [review]
Can you change all the descriptions to not start with: `"An image"

**Comment by forana** [comment]
> Can you change all the descriptions to not start with: `"An image"

Done.

**Pull request merged**